src/parameters.toml

```diff
@@ -38,6 +37,93 @@ value = 0.25
 type = "float"
 description = "Factor of relative contribution or Rgrowth (-)"
 
+
+# Farquhar Parameters
+
+[CO2_compensation_point_25c]
+value = 4.275e-5
+type = "float"
+description = "Γstar at 25 °C (mol/mol)"
+
+[CO2_michaelis_menten]
+value = 4.049e-4
+type = "float"
+description = "Michaelis-Menten parameter for CO2 at 25 °C (mol/mol)"
+
+[O2_michaelis_menten]
+value = 0.2874
+type = "float"
+description = "Michaelis-Menten parameter for O2 at 25 °C (mol/mol)"
+
+[CO2_activation_energy]
```

Comment:
For all activation energy numbers, can we site Bonan (2019), Table 11.2 and Bernacchi et al. (2001, 2003)?